crypto/x509/v3_ncons.c

```diff
@@ -628,8 +629,9 @@ static int nc_uri(ASN1_IA5STRING *uri, ASN1_IA5STRING *base)
     const char *hostptr = (char *)uri->data;
     const char *p = strchr(hostptr, ':');
     int hostlen;
+
     /* Check for foo:// and skip past it */
-    if (!p || (p[1] != '/') || (p[2] != '/'))
+    if (p == NULL || (p[1] != '/') || (p[2] != '/'))
```

Comment:
The parentheses should be done consistently - I do not like the unnecessary ones much but I'd be fine either with removing them or adding them to the first condition as well.

Reply:
On the one hand "this wasn't the point of this PR" on the other hand, I've said more than my share of "since you're here, please also fix..." :)  Fixed.